views/registration.njk

```diff
@@ -18,60 +18,66 @@
 
 {% block content %}
 <img src="/images/liability_terms.PNG" alt="Liability form">
-<form>
+<form method="POST">
   <div class="form-group row">
-    <label for="inputUserName" class="col-sm-2 col-form-label">Full name</label>
+    <label for="inputFirstName" class="col-sm-2 col-form-label">First name</label>
     <div class="col-sm-4">
-      <input class="form-control" id="inputUserName" placeholder="Enter full name" required>
+      <input type="firstName" name="firstName" class="form-control" id="inputFirstName" placeholder="Enter first name" required>
```

Comment:
Just FYI I think @bsm5 changed the names of these, so you may have to rename them here: https://github.com/LUCCA-Capstone/LUCCA/pull/34/files#diff-805eb7dfa0ef543601d731fc4449ed1aR101

Reply:
Yeah, he messaged me, I was waiting until his pull request got merged otherwise calls to the database fail. I guess it doesn't matter since there isn't a test on the "POST" route. I'll go ahead and update them now.